File: CODING/PYTHON/Pydroid3/fibonacci.py
a=0; b=1
t=()
t=t+(a,b)
n=int(input("enter the range of fibonacci no.:")) #fibonacci series
for i in range(n-2):
    c=a+b
    t=t+(c,)
    a=b
    b=c
print(t)    

        

File: CODING/c programing/Cxxdroid(non-exe)/recurssion/fibonacci.cxx
#include<stdio.h>
int fibonacci(int n)
{
     if (n<=1) return n;
     else return fibonacci(n-1)+fibonacci(n-2);
}
int main()
{
     int n;
     printf("enter the fibonacci series: ");
     scanf("%d", &n);
     for (int i=0; i<n;  i++)
     {
          printf("%d ", fibonacci(i));
     }
     return 0;
}